ha/core/controllers/cluster_controller.py

```diff
@@ -36,23 +36,31 @@ def initialize(self, controllers):
         raise HAUnimplemented("This operation is not implemented.")
 
     @controller_error_handler
-    def start(self) -> dict:
+    def start(self, sync=False, timeout=30) -> dict:
         """
         Start cluster and all service.
 
+        Args:
+            sync (bool, optional): if sync is True then start will check the status for timeout seconds.
+            timeout (int, optional): timeout(in seconds) can be specified for sync=True otherwise ignored.
+
         Returns:
-            ([dict]): Return dictionary. {"status": "", "msg":""}
+            ([dict]): Return dictionary. {"status": "", "output":"", "error":""}
```

Comment:
{"status": "", "output":"", "error":""}
need to update schema in ha/core/controllers/controller_annotation.py

Reply:
It's already done. Probably you missed in list of files.